db/compilers.xml

```diff
@@ -195,11 +195,14 @@
       <external>${PREFIX}gcc -v</external>
       <grep regexp="^[-\w]*gcc \S+ (\S+)" group="1"></grep>
     </variable>
+    <variable name="gcc_major_version">
+      <external>${PREFIX}gcc -dumpversion | cut -f1 -d.</external>
+    </variable>
     <runtimes default="default,kernel,native">
-       <directory group="default" >\.\./lib/gcc(-lib)?/$TARGET/$gcc_version/adalib/</directory>
-       <directory group="default" contents="^rts-">\.\./lib/gcc(-lib)?/$TARGET/$gcc_version/ada_object_path</directory>
-       <directory group="2" >\.\./lib/gcc(-lib)?/$TARGET/$gcc_version/rts-(.*)/adalib/</directory>
-       <directory group="1" >\.\./$TARGET/lib/gnat/(.*)/adalib/</directory>
+       <directory group="default" >\.\./lib*/gcc(-lib)?/$TARGET/($gcc_version|$gcc_major_version)/adalib/</directory>
```

Comment:
Why not `(64)?` rather than `*` which ensures we only match the pattern we know about?

Reply:
I did have (64)? initially, but then I though that there is no guarentee that some all system would use it.Some systems may have 64bits in /usr/lib and 32bits in /usr/lib32. We cannot tell.

Reply:
`(32|64)?` then? not fond of matching random stuff like libexec etc to be honest...